contrib/sbase/patches/cols.patch

```diff
@@ -0,0 +1,11 @@
+Index: sbase/sbase/cols.c
+===================================================================
+--- sbase.orig/sbase/cols.c	2022-12-08 22:33:35.296210736 +0100
++++ sbase/sbase/cols.c	2022-12-08 22:49:28.236179265 +0100
+@@ -1,5 +1,6 @@
+ /* See LICENSE file for copyright and license details. */
+ #include <sys/ioctl.h>
++#include <sys/ttycom.h>
```

Comment:
This manifests a problem with our header files. Please note that `sys/ioctl.h` in NetBSD includes `sys/ttycom.h` by default: https://mimiker.ii.uni.wroc.pl/source/xref/NetBSD/sys/sys/ioctl.h?r=53d22107#42